cookbooks/bcpc/recipes/powerdns.rb

```diff
@@ -28,13 +28,12 @@
 results = get_nodes_for("powerdns").map!{ |x| x['fqdn'] }.join(",")
 nodes = results == "" ? node['fqdn'] : results
 
+#
+# For some reason, we are compelled to specify a provider.
+# This will probably break if we ever move to chef-vault cookbook 2.x
```

Comment:
I haven't seen issues with Chef 12 and the latest chef-vault cookbook; why do you expect this to break?